internal/controllers/core/cluster/reconciler.go

```diff
@@ -239,9 +244,21 @@ func (c *connection) toStatus() v1alpha1.ClusterStatus {
 		connectedAt = &t
 	}
 
+	var reg *v1alpha1.RegistryHosting
+	if c.registry != nil {
+		reg = &v1alpha1.RegistryHosting{
+			Host:                   c.registry.Host,
+			HostFromClusterNetwork: c.registry.HostFromCluster(),
```

Comment:
ah, you reversed it! In the tilt internal data structure. "HostFromCluster" means the container runtime. see here:
https://github.com/tilt-dev/tilt/blob/master/internal/k8s/registry.go#L161

Reply:
(there was originally a single "host from cluster", but after a long conversation with the k8s sig-cluster-lifecycle team about it, we realized that there were two separate networks in the cluster - the network of the container runtime pulling the image, and the network inside the containers)

Reply:
ah, ok! updated.